package.json

```diff
@@ -116,9 +116,9 @@
     "@babel/core": "^7.20.0",
     "@babel/preset-env": "^7.20.0",
     "@babel/runtime": "^7.20.0",
-    "@formatjs/cli": "^6.2.0",
     "@comapeo/core": "1.0.1",
     "@comapeo/schema": "1.0.0",
+    "@formatjs/cli": "^6.2.0",
```

Comment:
This line moved (`npm install` alphabetizes the list) but the dependency didn't actually change.